tool/jt.rb

```diff
@@ -954,6 +956,42 @@ def test_ecosystem(env={}, *args)
   end
   private :test_ecosystem
 
+  def test_bundle(*args)
+    gems = [{:url => "https://github.com/sstephenson/hike", :commit => "3abf0b3feb47c26911f8cedf2cd409471fd26da1"}]
+    gems.each do |gem|
+      gem_url = gem[:url]
+      name = gem_url.split('/')[-1]
+      require 'tmpdir'
+      temp_dir = Dir.mktmpdir(name)
```

Comment:
In the block form, the temp directory is deleted at the end of the block. For testing, I want to frequently disable this by commenting out `# FileUtils.remove_entry temp_dir` to inspect the results. Also, in the future, I probably want to change this to not delete the temp directory whenever the test fails.